rest-api/src/main/java/gov/cms/qpp/conversion/api/services/CpcFileService.java

```diff
@@ -1,11 +1,14 @@
 package gov.cms.qpp.conversion.api.services;
 
 import gov.cms.qpp.conversion.api.model.UnprocessedCpcFileData;
+import java.io.InputStream;
 import java.util.List;
 
 /**
  * Service interface to handle processing cpc+ files
  */
 public interface CpcFileService {
 	List<UnprocessedCpcFileData> getUnprocessedCpcPlusFiles();
+
+	InputStream getFileById(String fileId);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Document this public method by adding an explicit description. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUndocumentedApi)